test/unit/staticWebsites.test.ts

```diff
@@ -219,4 +231,79 @@ describe("static websites", () => {
             },
         });
     });
+
+    it("should synchronize files to S3", async () => {
+        sinon.stub(CloudFormationHelpers, "getStackOutput").returns(Promise.resolve("bucket-name"));
```

Comment:
You should use `jest.mock` here instead -> https://jestjs.io/docs/mock-functions

Reply:
Couldn't make it work with Jest, e.g.

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/720328/119830584-b608d400-befc-11eb-8106-aa2dec3dedd2.png">

Reply:
manual `jest.mock` is to be done on the same scope level as the import (meaning outside of the describe block). If you want, you can merge it like this and I'll update one of the test with the corresponding jest mocking system to give you an exemple :)

Reply:
OK I see, but the problem above (in red) is the type error. But maybe I'm missing your point, I'll check out the example later.